message_definitions/v1.0/ras_a.xml

```diff
@@ -384,7 +384,7 @@
     </enum>
   </enums>
   <messages>
-    <message id="238" name="POI_REPORT">
```

Comment:
This one has to change - no one is allowed to use 0-256 as this is the original mavlink1 range, and is needed for the very rare cases where we need to have a mavlink 1 message. This is guaranteed to clash in future as there are not many unused values left.